src/services/config.js

```diff
@@ -22,6 +22,7 @@ class ConfigData {
     this.prompts = {};
     this.projects = [];
     this.activeProfile = null;
+    this.profiles = undefined;
```

Comment:
Needs to be instantiated as an object `{}` to avoid type mismatch later on while writing to config file

Reply:
Updated.